### PR TITLE
Book the elevator

### PR DESCRIPTION
**Pull request by djensenius**
Some small fixes to make the elevator booking possible. Extend the cookie timeout because it's annoying to login all the time.

### CODE REVIEW

File: src/services/admin-api.ts [hunk 1]
```diff
@@ -61,8 +61,13 @@ export interface ElevatorBooking {
   phoneDay: string;
   phoneNight: string
   deposit: number;
-  moveType: string;
+  moveType: number;
+  moveIn: boolean;
```
Comment (MaggieShan) [inline src/services/admin-api.ts:65]: Similar questions in condo-api PR review - is there a reason for not also including disposal/delivery bools?

File: src/managers/admin-api.ts [hunk 2]
```diff
@@ -68,6 +68,18 @@ export class AdminManager implements AdminApi {
     return deleteResult;
   }
 
+  public async approveElevatorBooking(id: number, formData: FormData): Promise<boolean> {
```
Comment (MaggieShan) [inline src/managers/admin-api.ts:71]: Should admins also have the ability to disapprove bookings?

Comment (djensenius) [reply src/managers/admin-api.ts:71]: I'm going to say no for now. The next step is some automated emails. If they approve and then change their mind they will need to reach out anyway.

File: src/services/admin-api.ts (hunk 1 above)
Comment (djensenius) [reply src/services/admin-api.ts:65]: Because I should delete delivery and disposal. 🤪

It's basically the same question (in, out, both?).

**Review by MaggieShan — APPROVED** [review]
lgtm

**Pull request merged**